# These parameters are supposed to be loade with the Track class
# There are two formats defined in here:
# the Standard format that you will receive from us;
# the Alternative format that is slightly different to make definitions by the obstacle editor easier.
# Both formats should behave identically if they are accessed through the Track class.
# For each format there is an example track. Both examples defines identical tracks.

# Standard format
# This is the format that you will receive for the finall competition.
# The format defines two named polyogons `stay_in` which the vehicle should
# stay inside of, and keep_out, which the vehicle should not enter.
# Each polygon is defined as a list of points.
# Each point is a list with two coordinates, x and y.
# The polygons that you will get from us can be assumed to be non self intersecting.
# The polygons will NOT be guaranteed to be convex.
# The polygons as defined here will not necesarily be closed (as in polygon[0] == polygon[-1]).
# The polygons you get from the Track class's `stay_in` and `keep_out` is however guaranteed to be closed.

stay_in:
- [7.062383651733398, 6.030817031860352]
- [10.639168739318848, 10.9046049118042]
- [11.220185279846191, 11.79182243347168]
- [10.126347541809082, 12.529036521911621]
- [6.63116455078125, 15.012391090393066]
- [5.266538619995117, 15.254671096801758]
- [3.251222610473633, 12.287466049194336]
- [1.1301617622375488, 9.165960311889648]
- [0.8839321136474609, 6.052201271057129]
- [0.8474733829498291, 4.697922706604004]
- [-0.10617148876190186, 4.062297821044922]
- [-2.512756586074829, 3.633636951446533]
- [-3.0355911254882812, 3.3094892501831055]
- [-4.974943161010742, 0.39020490646362305]
- [-7.031565189361572, -2.586428165435791]
- [-8.239501953125, -4.5739054679870605]
- [-6.335666656494141, -5.958523273468018]
- [-4.8640594482421875, -6.835036277770996]
- [-3.769484043121338, -8.087359428405762]
- [-8.11494255065918, -14.977669715881348]
- [-8.169170379638672, -15.092684745788574]
- [-6.833065986633301, -15.777475357055664]
- [-5.471351146697998, -13.424124717712402]
- [-4.823225021362305, -12.010622024536133]
- [-2.1649327278137207, -7.864295959472656]
- [3.7034802436828613, 0.9685764312744141]
- [2.9949851036071777, 2.9068355560302734]
- [3.6532697677612305, 3.789060592651367]
- [5.797752380371094, 3.9103240966796875]
keep_out:
- [-2.257740020751953, -5.963809490203857]
- [-0.051998019218444824, -2.6962413787841797]
- [2.177204132080078, 0.8427577018737793]
- [1.4386078119277954, 3.1998801231384277]
- [2.9324440956115723, 5.283330917358398]
- [5.111310958862305, 5.297026634216309]
- [6.743153095245361, 7.399977684020996]
- [9.46615982055664, 11.50140380859375]
- [5.988212585449219, 13.953290939331055]
- [2.969768762588501, 9.814141273498535]
- [2.064190149307251, 8.308368682861328]
- [2.8221001625061035, 5.632914066314697]
- [1.1833834648132324, 3.105790138244629]
- [-1.5220427513122559, 1.8044390678405762]
- [-2.3908848762512207, 1.9549970626831055]
- [-4.520245552062988, -1.2392582893371582]
- [-5.999081611633301, -3.434451103210449]

# Alternative format, can be useful for testing with the obstacle editor
# This format defines a list, named `track`, containing exactly two polygons.
# The first polygon is the stay in polygon.
# The second polygon is the keep out polygon.
# In all other aspects this format shoukd be identical to the Standard format.
track:
- [[7.062383651733398, 6.030817031860352], [10.639168739318848, 10.9046049118042], [11.220185279846191, 11.79182243347168], [10.126347541809082, 12.529036521911621], [6.63116455078125, 15.012391090393066], [5.266538619995117, 15.254671096801758], [3.251222610473633, 12.287466049194336], [1.1301617622375488, 9.165960311889648], [0.8839321136474609, 6.052201271057129], [0.8474733829498291, 4.697922706604004], [-0.10617148876190186, 4.062297821044922], [-2.512756586074829, 3.633636951446533], [-3.0355911254882812, 3.3094892501831055], [-4.974943161010742, 0.39020490646362305], [-7.031565189361572, -2.586428165435791], [-8.239501953125, -4.5739054679870605], [-6.335666656494141, -5.958523273468018], [-4.8640594482421875, -6.835036277770996], [-3.769484043121338, -8.087359428405762], [-8.11494255065918, -14.977669715881348], [-8.169170379638672, -15.092684745788574], [-6.833065986633301, -15.777475357055664], [-5.471351146697998, -13.424124717712402], [-4.823225021362305, -12.010622024536133], [-2.1649327278137207, -7.864295959472656], [3.7034802436828613, 0.9685764312744141], [2.9949851036071777, 2.9068355560302734], [3.6532697677612305, 3.789060592651367], [5.797752380371094, 3.9103240966796875]]
- [[-2.257740020751953, -5.963809490203857], [-0.051998019218444824, -2.6962413787841797], [2.177204132080078, 0.8427577018737793], [1.4386078119277954, 3.1998801231384277], [2.9324440956115723, 5.283330917358398], [5.111310958862305, 5.297026634216309], [6.743153095245361, 7.399977684020996], [9.46615982055664, 11.50140380859375], [5.988212585449219, 13.953290939331055], [2.969768762588501, 9.814141273498535], [2.064190149307251, 8.308368682861328], [2.8221001625061035, 5.632914066314697], [1.1833834648132324, 3.105790138244629], [-1.5220427513122559, 1.8044390678405762], [-2.3908848762512207, 1.9549970626831055], [-4.520245552062988, -1.2392582893371582], [-5.999081611633301, -3.434451103210449]]

